File: c/numeros_pares.c
#include <stdio.h>

int main(){
    int n, qtdpares;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    int vetor[n];

    for (int i=0; i<n; i++) {
        printf("Digite um numero: ");
        scanf("%d", &vetor[i]);
    }

    printf("\nNUMEROS PARES:\n");

	qtdpares = 0;
    for (int i=0; i<n; i++) {
        if (vetor[i] % 2 == 0) {
            printf("%d  ", vetor[i]);
            qtdpares++;
        }
    }

    printf("\n\nQUANTIDADE DE PARES = %d\n", qtdpares);

    return 0;
}

File: java/numeros_pares.java
import java.util.Scanner;

public class numeros_pares {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
		
		int n, qtdpares;

	    System.out.print("Quantos numeros voce vai digitar? ");
	    n = sc.nextInt();

	    int[] vetor = new int[n];

	    for (int i=0; i<n; i++) {
	    	System.out.print("Digite um numero: ");
	        vetor[i] = sc.nextInt();
	    }

	    System.out.println("\nNUMEROS PARES:");

		qtdpares = 0;
	    for (int i=0; i<n; i++) {
	        if (vetor[i] % 2 == 0) {
	        	System.out.printf("%d  ", vetor[i]);
	            qtdpares++;
	        }
	    }

	    System.out.printf("\n\nQUANTIDADE DE PARES = %d\n", qtdpares);

		sc.close();
	}
}
